### DISCUSSION TITLE
change icon on cTKTopLevel?

### DISCUSSION THREAD

**Discussion by Borgelin** [TomSchimansky/CustomTkinter#1607] [Q&A]
Hi,

I've made an app and in the main window I have a custom icon, works fine without any problems.
But when a CTkTopLevel-window shows, I cannot get it to use the xxx.ico-file.

Here is my code for the top level:

```
def show_please_wait(self):
  self.please_wait_window = ctk.CTkToplevel(self)
  self.please_wait_window.title("xxxxxxx")
  self.please_wait_window.geometry("400x100")
  self.please_wait_window.iconbitmap('xxx.ico')
  self.please_wait_window.transient(self)
  self.please_wait_window.grab_set()
  label = ctk.CTkLabel(self.please_wait_window, text="xxxxxxxx")
  label.pack(padx=20, pady=20)
```

Any suggestions?

**Comment by avalon60** [comment]
Maybe worth checking  [#1163](https://github.com/TomSchimansky/CustomTkinter/issues/1163). Looks similar.
